docs/guides/getting-started/get-an-account.rst

```diff
@@ -90,14 +90,14 @@ Getting an account on the RSP
         If you publish as *Lady Gaga*, don't sign up as *Stefani Joanne Angelina Germanotta*.
 
       - Provide any email you want provided you can immediately access it (for the confirmation).
-        Additionally, using your institutional email address (instead of say, a Gmail account) helps our verification process; please do so if you can.
+        Additionally, using your main institutional email address (instead of say, a Gmail account) helps our verification process; please do so if you can.
 
-   6. Email confirmation step:
+   6. Onboarding flow: Email confirmation step
 
       -  You will receive an email to the address you provided.
          Please look out for it and click on the link inside it to complete the process.
 
-   7. Finishing steps:
+   7. Onboarding flow: Finishing steps:
```

Comment:
I'd rather not have a double colon; a colon is there to introduce a sublist, the second use is trying to add additional pseudo-hierarchy to this list. Maybe do an em-dash instead, like

```suggestion
   7. Onboarding flow — Finishing steps:
```

But honest, all these onboarding flows just add clutter an add complexity to the document. Do we really need this?